modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -32,8 +37,26 @@ networking:
   - 172.30.0.0/16
 platform:
   none: {} <10>
+ifndef::restricted[]
 pullSecret: '{"auths": ...}' <11>
+endif::restricted[]
+ifdef::restricted[]
+pullSecret: '{"auths":{"<bastion_host_name>:5000": {"auth": "<credentials>","email": "you@example.com"}}}' <11>
+endif::restricted[]
 sshKey: 'ssh-ed25519 AAAA...' <12>
+ifdef::restricted[]
+additionalTrustBundle: | <13>
+  -----BEGIN CERTIFICATE-----
+  ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+  -----END CERTIFICATE-----
+imageContentSources: <14>
+- mirrors:
+  - <bastion_host_name>:5000/<repo_name>/release
+  source: quay.io/openshift-release-dev/ocp-release
+- mirrors:
+  - <bastion_host_name>:5000/<repo_name>/release
+  source: registry.svc.ci.openshift.org/ocp/release
```

Comment:
Both examples of mirrors are the same, we need to provide different options on each mirror example or left just one